docs/js/features/odata/odata-v4-client.md

```diff
@@ -19,4 +19,30 @@ Typed client for OData v4 is under heavy development and was released only as `e
 
 ## Making a request using a generated OData v4 client
 
-Be patient, we'll soon provide documentation on how to use the SAP Cloud SDK OData v4 clients.
+Be patient, we are working on the it.
+
+### Filter
+
+##### Filter on One-To-Many Navigation Properties
+
+OData V4 introduces the [lambda operators](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_LambdaOperators) e.g., `any`/`all`, so that the root property of the one-to-many navigation properties can be filtered. Below is an example that demonstrates how to use the lambda operator [any](/cloud-sdk/api/1.28.0/modules/sap_cloud_sdk_core#any)) as a filter.
```

Comment:
```suggestion
OData V4 introduces the [lambda operators](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_LambdaOperators) e.g., `any`/`all`, so that the root property of the one-to-many navigation properties can be filtered. Below is an example that demonstrates how to use the lambda operator [any](/cloud-sdk/api/1.28.0/modules/sap_cloud_sdk_core#any)).
```